python/util/reactjs.py

```diff
@@ -39,14 +39,20 @@
         var lintOptions = {{
             "globals": {{
              "CURRENCY": false,
+             "PRODUCT_CHECKOUT_FORM": false,
              "React": false,
+             "ReactDOM": false,
              "URL_MAP": false,
+             "UTILITIES_JSX": false,
              "moment": false,
              "toastSettingsJobProductionAssignmentComments": false,
              "toastSettingsJobProductionAssignmentQuote": false,
              "$": false,
              "MERCHI": false,
-             "console": false
+             "console": false,
+             "job": false,
+             "currentUser": false,
+             "window": false,
```

Comment:
you've only added `job` to props, so what's all this other unrelated stuff doing here?

Reply:
- PRODUCT_CHECKOUT_FORM I've imported the Address component from
- window to redraw the pdf.
- ReactDOM should be added for future development
- UTILITIES_JSX I've imported alertTemplate
- currentUser I've removed.